docs/integrations/neutron.mdx

```diff
@@ -13,13 +13,11 @@ title: Neutron
 
    The best way to get the Slinky Sidecar binary is in the GitHub releases page for Slinky.
 
-   https://github.com/skip-mev/slinky/releases/tag/v1.0.0
+   https://github.com/skip-mev/slinky/releases/tag/v1.0.1
```

Comment:
Probably best not to link to a specific release here since it will easily fall out of date.